### DISCUSSION TITLE
Script signature verification failed

### DISCUSSION THREAD

**Discussion by ellingtonsu** [CoolBitX-Technology/coolwallet-signing-tools#50] [Q&A] [answered]
- Description
While testing Polkadot example in script tester, script signature verification failed

- Expected Behavior
After entering script and argument directly,  a signature should be generated

- Screenshot
![script_test](https://user-images.githubusercontent.com/4967808/153359412-23180809-7fc8-4118-9276-81ac6fa88568.png)

**Comment by hankliu418** [comment, marked as answer]
Would you please update card via script tester to make sure the card is develop card, thx!

**Reply by ellingtonsu** [reply]
Is Token required when updating the card? since update error if Token is empty. If it is, what Token should I enter? Thanks~
